src/CreateEventQueryView/CreateEventQueryView.js

```diff
@@ -68,19 +96,19 @@ class CreateEventQueryView extends ConnectionComponent {
 		window.history.back();
 	}
 
-	render() {
-		const allFetches = PromiseState.all([this.props.eventType, this.props.attributes]);
-		const optionalActions = this.props.createEventQueryResponse;
-		if(optionalActions && optionalActions.fulfilled) {
-			window.history.back();
-			return null;
-		}
-		const eventType = this.props.eventType.value;
-		const attributes = this.props.attributes.value;
-		const connectionIncomplete = super.render(allFetches);
-		if (connectionIncomplete) {
-			return connectionIncomplete;
-		}
+	renderCreateEventQuery() {
```

Comment:
In `renderCreateEventQuery` and `renderEditEventQuery` there is way too much code duplication.
A parameterized function or splitting it into more, smaller functions might do the trick.